cluster/v1beta1/types_placement.go

```diff
@@ -215,6 +221,49 @@ type AddOnScore struct {
 	ScoreName string `json:"scoreName"`
 }
 
+// SpreadPolicy defines how the placement decision should be spread among the ManagedClusters.
+type SpreadPolicy struct {
+	// SpreadConstraints defines how the placement decision should be distributed among a set of ManagedClusters.
+	// The importance of the SpreadConstraintsTerms follows the natural order of their index in the slice.
+	// The scheduler first consider SpreadConstraintsTerms with smaller index then those with larger index
+	// to distribute the placement decision.
+	// +optional
+	// +kubebuilder:validation:MaxItems=8
+	SpreadConstraints []SpreadConstraintsTerm `json:"spreadConstraints,omitempty"`
+}
+
+// SpreadConstraintsTerm defines a terminology to spread placement decisions.
+type SpreadConstraintsTerm struct {
+	// TopologyKey is either a label key or a cluster claim name of ManagedClusters.
+	// +required
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:Pattern=`^([a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*/)?([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9]$`
+	// +kubebuilder:validation:MaxLength=316
+	TopologyKey string `json:"topologyKey"`
+
+	// TopologyKeyType indicates the type of TopologyKey. It could be Label or Claim.
+	// +required
+	// +kubebuilder:validation:Required
+	TopologyKeyType TopologyKeyType `json:"topologyKeyType"`
+
+	// MaxSkew describes the degree to which the workload may be unevenly distributed.
+	// Skew is the maximum difference between the number of selected ManagedClusters in a topology and the global minimum.
+	// The global minimum is the minimum number of selected ManagedClusters for the topologies within the same TopologyKey.
+	// The minimum possible value of MaxSkew is 1.
```

Comment:
add default. should we also add a Maximum?

Reply:
After thinking about it, I decided to set maxSkew to a nullable field to differentiate the different behaviors of the scheduler, which is different from what we have discussed earlier. Hence, it may not have a default value (or, the default value is nil). The [updated comments](https://github.com/open-cluster-management-io/api/blob/6c01801356ce0cff7174e10816c32ab1cdb49d8d/cluster/v1beta1/types_placement.go#L254-L259) and [this section](https://github.com/open-cluster-management-io/enhancements/blob/9873893f1ec04cf7c84fff23e43ee05363c66b32/enhancements/sig-architecture/70-spread-policy/README.md?plain=1#L276-L299) in the proposal describes its semantics.

The maxSkew is a maximum threshold of the skew, which may not be reached. I think it may not have a maximum value.